subjects/mobile-dev/chess/resources/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.19.1-buster AS builder
+
+WORKDIR /app
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+COPY . .
+RUN go build -o /app/main .
+
+
+FROM ubuntu:20.04
```

Comment:
I have commented from line 11 to line 14 and I still get the same result. 

I don't understand the value of having this additional lines in the Dockerfile

Reply:
will close this until we work on it!